benchmarks/writer.cpp

```diff
@@ -39,20 +39,18 @@ void writer(int cols, int rows)
 
     for(int index = 0; index < rows; index++)
     {
-        if ((index + 1) % (rows / 10) == 0)
```

Comment:
tracking could really be done in a more reliable way. Current setup is buggy and doesn't account for serialisation at all